chasm/src/testData/results/other/test_local_variables.result

```diff
@@ -56,15 +56,15 @@ public class other/TestLocalVariables {
    L5
     LOCALVARIABLE param1 I L0 L5 0
     LOCALVARIABLE param2 D L0 L5 1
-    LOCALVARIABLE local1 I L1 L5 3
-    LOCALVARIABLE local2 D L3 L5 4
+    LOCALVARIABLE local1 I L0 L5 3
+    LOCALVARIABLE local2 D L2 L4 4
```

Comment:
I was just looking through this change. Is there a reason the the start labels are decreased by one? This doesn't seem intentional.